src/main/resources/db/migration/V15__20251117__create_mentees_data_model.sql

```diff
@@ -0,0 +1,48 @@
+-- Table for mentees
+CREATE TABLE IF NOT EXISTS mentees
+(
+    mentee_id        INTEGER PRIMARY KEY REFERENCES members (id) ON DELETE CASCADE,
+    profile_status   INTEGER NOT NULL         DEFAULT 4,
+    bio              TEXT    NOT NULL,
+    years_experience INTEGER                  DEFAULT 0,
+    spoken_languages VARCHAR(200),
+    created_at       TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+    updated_at       TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT fk_profile_status FOREIGN KEY (profile_status)
+        REFERENCES member_statuses (id)
+);
+
+
+-- TABLE Mentee mentorship types (join table)
+CREATE TABLE IF NOT EXISTS mentee_mentorship_types
+(
+    mentee_id       INTEGER NOT NULL REFERENCES mentees (mentee_id) ON DELETE CASCADE,
+    mentorship_type INTEGER NOT NULL REFERENCES mentorship_types (id) ON DELETE CASCADE,
+    PRIMARY KEY (mentee_id, mentorship_type)
+);
+
+-- TABLE Mentee previous mentorship types (join table)
+CREATE TABLE IF NOT EXISTS mentee_previous_mentorship_types
```

Comment:
in this case we don't need to store the information again you can re-use from previous table. Adding the year solve the problem and this table is not necessary.